metadata/metadata.go

```diff
@@ -197,20 +197,19 @@ func FromIncomingContext(ctx context.Context) (MD, bool) {
 
 // ValueFromIncomingContext returns value from the incoming metadata if exists.
 //
-// This is intended for using as fast access to context value with string constant.
+// ValueFromIncomingContext returns the metadata value corresponding to the metadata
```

Comment:
Delete comment above: "ValueFromIncomingContext returns value from the incoming metadata if exists."